crates/cheatnet/src/cheatcodes/precalculate_address.rs

```diff
@@ -17,11 +14,12 @@ impl CheatnetState {
         class_hash: &ClassHash,
         calldata: &[Felt252],
     ) -> ContractAddress {
-        let account_address = ContractAddress(patricia_key!(TEST_ACCOUNT_CONTRACT_ADDRESS));
         let salt = self.get_salt();
 
         let execute_calldata = create_execute_calldata(calldata);
-        calculate_contract_address(salt, *class_hash, &execute_calldata, account_address).unwrap()
+        let deployer_address =
+            Felt252::from(0x0000_1724_9872_3497_3219_3472_1083_7402_i128).to_contract_address();
```

Comment:
I believe it also should be extracted as a const

Reply:
done